core/services/versionchooser/utils/dockerhub.py

```diff
@@ -76,10 +76,13 @@ async def fetch_sha(self, metadata: TagMetadata) -> str:
             async with session.get(
                 f"{self.index_url}/v2/{metadata.repository}/manifests/{metadata.digest}", headers=header
             ) as resp:
+                data = await resp.json(content_type=None)
                 if resp.status != 200:
-                    warn(f"Error status {resp.status}")
+                    warn(f"Error status {resp.status} {data}")
+                    # Sha only exist locally
+                    if data["errors"].len() == 1 and data["errors"][0]["code"] == "MANIFEST_UNKNOWN":
```

Comment:
```suggestion
                    if len(data["errors"]) == 1 and data["errors"][0]["code"] == "MANIFEST_UNKNOWN":
```
<img width="339" alt="Screenshot 2023-01-22 at 11 08 46" src="https://user-images.githubusercontent.com/4013804/213920153-37fa83a6-8aa2-483d-918c-0f10be7b8d6d.png">

Reply:
Ops.